notebooks/official/pipelines/google_cloud_pipeline_components_automl_tabular.ipynb

```diff
@@ -353,9 +353,9 @@
         "id": "timestamp"
```

Comment:
<p>Drop the bullet list.</p><p> </p><p>Otherwise, you can delete the individual resources you created in this tutorial --&nbsp;<em>Note:</em>&nbsp;this is auto-generated and not all resources may be applicable for this tutorial: -&gt;</p><p> </p><p>Otherwise, you can delete the individual resources you created in this tutorial .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>